src/Columns/ColumnLowCardinality.cpp

```diff
@@ -311,6 +311,11 @@ void ColumnLowCardinality::compareColumn(const IColumn & rhs, size_t rhs_row_num
             compare_results, direction, nan_direction_hint);
 }
 
+bool ColumnLowCardinality::hasEqualValues() const
+{
+    return getDictionary().size() <= 1;
```

Comment:
Well, actually, this is not correct.
It is possible that not all rows of dictionary are used.
We still need to check that columns with indexes has same value.

Reply:
Ok...